tests/bundle.yaml

```diff
@@ -1,11 +0,0 @@
-bundle: kubernetes
-applications:
-  kubeflow-dashboard:
-    charm: cs:kubeflow-dashboard
-    scale: 1
-  kubeflow-profiles:
-    charm: cs:kubeflow-profiles
-    source: ../
-    scale: 1
-relations:
-  - [kubeflow-profiles, kubeflow-dashboard]
```

Comment:
There was discussion last week on whether it made sense to put the logic for deploying things needed for a test in a bundle yaml vs in the pytest tests.  Doesn't matter for this particular charm, but we should discuss and see if using one or the other consistently everywhere makes sense.  I think the basics of the previous discussion were that if we can use yaml + pytest_operator's `render_bundle` it might result in cleaner tests, but that there might have been a few features to add in the operator to get things running right

Reply:
It might be interesting for bigger tests, but I think it is a bit unnecessary for a 2 charm deployment.

Reply:
yeah I kinda agree with you, but it doesn't take much away either and being consistent across our charms might be interesting.  It would be very clean to have a pattern of:
```
charm_path = build(this_charm)
rendered_bundle = render_bundle(bundle_file, charm=charm_path)
deploy(rendered_bundle)
```
rather than having the individual calls.  That would make most of our tests be identical, with the only differences being in the bundle yaml (some "bundles" could be just the single charm, others be complex)

Either way it doesn't matter for this PR